Robot Code/src/org/usfirst/frc/team3316/robot/sequences/ClimbingSequence.java

```diff
@@ -0,0 +1,17 @@
+package org.usfirst.frc.team3316.robot.sequences;
+
+import org.usfirst.frc.team3316.robot.commands.DBugCommandGroup;
+import org.usfirst.frc.team3316.robot.commands.climbing.CloseArmPiston;
+import org.usfirst.frc.team3316.robot.commands.climbing.MoveUp;
+import org.usfirst.frc.team3316.robot.commands.climbing.WaitForRung;
+
+public class ClimbingSequence extends DBugCommandGroup
+{
+	
+	public ClimbingSequence() {
+		addSequential(new CloseArmPiston());
+		addSequential(new WaitForRung());
```

Comment:
Make this interruptable in case we don't detect the rung properly, and the driver will want to initialize the climb anyway.